regulations/static/regulations/css/less/module/comment-confirm.less

```diff
@@ -0,0 +1,42 @@
+
+.comment-confirm-success {
+  .font-regular;
+  padding: 30px 20px;
+  width: 80%;
+
+  .success-alert {
+    background-color: @green_light;
+    display: -webkit-flex;
+    display: flex;
```

Comment:
[Looks](http://caniuse.com/#feat=flexbox) like IE has significant issues with `display: flex`. Does it give us much?